releaseNotes/KubernetesProvider.md

```diff
@@ -2,6 +2,19 @@
 
 [Image][image] | [Sample][sample]
 
+## 1.1.0 - xx xx, 2023

```

Comment:
is this date intentional?

Reply:
Yes, the final date will be updated before provider release.